### DISCUSSION TITLE
Not getting nvm major updates

### DISCUSSION THREAD

**Discussion by guidojw** [renovatebot/renovate#12438] [Request Help] [answered]
### How are you running Renovate?

WhiteSource Renovate hosted app on github.com

### Describe the bug

I'm not getting nvm major updates in my project, minor updates do work and are getting grouped correctly.

Minimal reproduction repository:
https://github.com/guidojw/renovate-test

I have the feeling a simple configuration change might fix this, but tried a lot of changes and haven't found what yet.

If I understand the job logs correctly, a major version is not received from the github-tags datasource and that's why it's not included. The docker datasource however does include a major version and on the node repository tag 17.0.1 exists so that confuses me.

### Relevant debug logs

<details><summary>Logs</summary>

```
"nvm": [
      {
        "packageFile": ".nvmrc",
        "deps": [
          {
            "depName": "node",
            "currentValue": "16.6.0",
            "datasource": "github-tags",
            "lookupName": "nodejs/node",
            "depIndex": 0,
            "updates": [
              {
                "bucket": "non-major",
                "newVersion": "v16.13.0",
                "newValue": "16.13.0",
                "releaseTimestamp": "2021-10-26T16:22:38.000Z",
                "newMajor": 16,
                "newMinor": 13,
                "updateType": "minor",
                "branchName": "renovate/node.js-(non-major)"
              }
            ],
            "warnings": [],
            "versioning": "node",
            "sourceUrl": "https://github.com/nodejs/node",
            "currentVersion": "16.6.0",
            "isSingleVersion": true,
            "fixedVersion": "16.6.0"
          }
        ]
      }
    ]
```

</details>

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction repository in the bug description

**Comment by viceice** [comment, marked as answer]
By default renovate will consider only node lts as stable.

**Reply by guidojw** [reply]
With the package rules in the configuration I was hoping to get non-major and major updates for Dockerfile + nvm in separate PRs, is there another way of achieving this?

**Reply by viceice** [reply]
Do you like to have the unstable v17 update? like  https://github.com/guidojw/renovate-test/pull/3

Or do you only like to have node LTS updates?

**Reply by guidojw** [reply]
I would like to have the unstable updates too yes, so like that PR (but that only updates Node in the Dockerfile).

Similar to how I'm currently handling package.json dependency updates; major and non-major updates in their own PRs.

**Reply by rarkins** [reply]
You may be interested in `ignoreUnstable=false` (will allow v17 updates) and potentially `separateMultipleMajor=true`

**Reply by guidojw** [reply]
`ignoreUnstable: false` did the thing! Thanks :)

I have one question about your other suggestion:
If I already have a v16.x->v17.x PR open and a newer v17 is released, will it create a separate PR for that too?
Or is my initial understanding correct that it overwrites the existing PR and only creates new PRs for updates like v16->v18, whenever that comes out and the old PR isn't merged yet.

**Reply by rarkins** [reply]
For major updates, it overwrites the existing PR (e.g. from 17.0.0 to 17.1.0). If v18 came out and you still had v17 open, by default it would autoclose v17 and open v18. But if you have separateMultipleMajor=true then it would keep v17 open and updating until you merged it or v18, or updated manually such that it's no longer needed.

**Reply by guidojw** [reply]
All right, thanks for the clarification!
